patterns/dswp-hero-image-with-title.php

```diff
@@ -0,0 +1,33 @@
+        <div class="wp-block-group" style="margin-top:0;margin-bottom:0"><!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|30","bottom":"var:preset|spacing|50","left":"var:preset|spacing|60","right":"var:preset|spacing|60"},"margin":{"top":"0","bottom":"0"}},"dimensions":{"minHeight":"278px"},"border":{"radius":"4px","left":{"color":"var:preset|color|border-dark","width":"6px"},"top":{},"right":{},"bottom":{}}},"backgroundColor":"background-white","layout":{"type":"default"}} -->
+            <div class="wp-block-group has-background-white-background-color has-background" style="border-radius:4px;border-left-color:var(--wp--preset--color--border-dark);border-left-width:6px;min-height:278px;margin-top:0;margin-bottom:0;padding-top:var(--wp--preset--spacing--30);padding-right:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--60)"><!-- wp:group {"style":{"spacing":{"padding":{"right":"0","left":"0","top":"0","bottom":"0"},"margin":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|30"}}},"layout":{"type":"constrained"}} -->
+                <div class="wp-block-group" style="margin-top:var(--wp--preset--spacing--50);margin-bottom:var(--wp--preset--spacing--30);padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><!-- wp:heading {"textAlign":"left","style":{"elements":{"link":{"color":{"text":"var:preset|color|font dark"}}},"spacing":{"padding":{"top":"0","bottom":"0"},"margin":{"top":"0","bottom":"0"}}},"textColor":"font dark","fontSize":"large"} -->
+                    <h2 class="wp-block-heading has-text-align-left has-font-dark-color has-text-color has-link-color has-large-font-size" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0">Transportation Investment Corporation</h2>
+                    <!-- /wp:heading -->
+                </div>
+                <!-- /wp:group -->
+
+                <!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"default"}} -->
+                <div class="wp-block-group" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:paragraph {"style":{"elements":{"link":{"color":{"text":"var:preset|color|font dark"}}},"spacing":{"margin":{"top":"0","bottom":"0"},"padding":{"top":"0","bottom":"0"}},"typography":{"lineHeight":"1.8"}},"textColor":"font dark","fontSize":"bcds-large-body"} -->
+                    <p class="has-font-dark-color has-text-color has-link-color has-bcds-large-body-font-size" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0;line-height:1.8">TI Corp is currently delivering the Pattullo Bridge Replacement, the Broadway Subway Project, the Fraser River Tunnel Project and Steveston Interchange Project (components of the Highway 99 Tunnel Program), and the Surrey Langley SkyTrain.</p>
```

Comment:
I thought we were removing all site specific information (ticorp) form the design system theme